src/io/node_http_test.ts

```diff
@@ -114,8 +132,12 @@ describe('nodeHTTPRequest-load', () => {
     expect(new Float32Array(modelArtifacts.weightData)).toEqual(floatData);
 
     expect(requestInits).toEqual([
-      {credentials: 'include', cache: 'no-cache'},
-      {credentials: 'include', cache: 'no-cache'}
+      {credentials: 'include', cache: 'no-cache', headers: {Accept: JSON_TYPE}},
```

Comment:
we are removing the Accept header from the request.

Reply:
In the future or for 0.15.0?